### ISSUE TITLE
ArgenMap Topográfico sin etiquetas

### ISSUE THREAD

**Issue by gasato** [ign-argentina/argenmap#243]
Hola, el producto ArgeMap Topográfico es muy bueno, pero no podrían hacer otra versión sin las etiquetas?
El tema es que a veces prefiero poner yo mismo las etiquetas porque en general en mi QGIS se ven muy chiquitas o quiero poner sólo las que necesito.

Saludos, gracias Gabriel

**Comment by pablodegrande** [comment]
sí, coincido en que esto sería muy útil... hay veces que los usuarios precisan ver la forma de las cosas, sin las etiquetas superpuestas

**Comment by Zalitoar** [comment]
Gracias por las sugerencias. Este repositorio es del visor de mapas, que no define ni tiene relación directa con los mapas base que muestran sin embargo acercaremos sus sugerencias.
Con respecto a cómo se ve un mapa base en QGIS, una vez acercada la vista a la posición y zoom deseados, hacer click derecho sobre el mapa base activo y luego en la opción "Zoom a Resolución Nativa (100%)" que ajusta el mapa a la vista actual.
